MANIFESTO.md

```diff
@@ -3,37 +3,37 @@ layout: page
 
-The availability of knowledge and skilled engineers is the limiting factor for many organizations to adopt, leverage, and successfully operate complex technology. For true self-determination in the ever-growing digital domain, we need to build skills and share the gained knowledge freely and in unlimited ways also in the realm of operational knowledge — exactly the way we started with software code almost half a decade ago. We encourage organizations to collectivize operational knowledge and share it with the entire Open Operations community.
+The availability of knowledge and skilled engineers is the limiting factor for many organizations to adopt, leverage, and successfully operate complex technology. For true self-determination in the ever-growing digital domain, we need to build skills and share the gained knowledge freely and in unlimited ways, including in the realm of operational knowledge — which is exactly the way we started with software code almost half a decade ago. We encourage organizations to collectivize operational knowledge and share it with the entire Open Operations community.
 
 ## [We](#supporting-organizations) are transparent about our incidents
 
-We firmly believe that failures make us experts. The way we handle mistakes is how we become better. We think that handling mistakes in an optimal way means to communicate about them openly and without shame. We are convinced that the culture we build around failures is fundamental for an innovative environment: Not being afraid of making mistakes opens the door to trying new things, without having the fear to fail. We encourage people to raise their voice - not only in case of success, but also in case of failure. We are sure that this is one of the best ways to prevent ourselves and others from making the same mistakes. To achieve the state of complete transparency, the history of failures and incidents should be recorded and preserved. Our goal is that others can truly experience trust in this way, rather than just being told they can.
+We firmly believe in the princple of learning from our mistakes and that lessons learnt add to our expertise. The way we handle errors is how we become better. We feel that handling our mistakes in the best way possible means communicating about them openly and without shame. We are convinced that the culture we build around failures is fundamental for an innovative environment: not being afraid of making mistakes opens the door to trying out new approaches, without fear of failing. We encourage people to speak out – not just when they are successful, but also when they fail. We are sure that this is indeed one of the best ways to prevent ourselves and others from making the same mistakes. We hold that to achieve the state of complete transparency, the history of failures and incidents should be recorded and preserved. Our goal is that others can truly experience trust in this way, rather than just being told they can.
```

Comment:
I'm saddened to see the sentence "failures make us experts" being removed and fail to see the reason why this - in my opinion - important way of stating something in a very explicit manor needs to go.

Reply:
We are not obliged to accept all changes. Maybe we just stick to the "failures makes us experts" wording (which I do like). Or maybe just ask James why he proposed this change?

Reply:
@itrich please check the change from 84af578. Maybe this is a good way?